### PR TITLE
Fixes #352 Vignettes: fix typos and add TOC 

### LINKED ISSUE #352: Vignettes: fix typos and add TOC

### LINKED ISSUE #353: Vignettes: Define report output style

### PR DESCRIPTION
**Pull request by Yuri05**
Fixes #353 Vignettes: Define report output style

### CODE REVIEW

File: vignettes/goodness-of-fit.Rmd [hunk 1]
```diff
@@ -3,12 +3,16 @@ title: "Time profiles and Residuals task"
 output: 
   rmarkdown::html_vignette:
    number_sections: true
+   toc: true
```
Comment (Yuri05) [inline vignettes/goodness-of-fit.Rmd:6]: Adding table of contents for every vignette

File: vignettes/goodness-of-fit.Rmd [hunk 2]
```diff
@@ -3,12 +3,16 @@ title: "Time profiles and Residuals task"
...
    number_sections: true
+   toc: true
 vignette: >
   %\VignetteIndexEntry{goodness-of-fit}
   %\VignetteEngine{knitr::rmarkdown}
   %\VignetteEncoding{UTF-8}
 ---
 
+```{r report output style, child="reportOutputStyle.css", eval=TRUE}
+```
```
Comment (Yuri05) [inline vignettes/goodness-of-fit.Rmd:14]: Style to be used for every report output within the vignette

File: vignettes/goodness-of-fit.Rmd [hunk 3]
```diff
@@ -139,9 +143,11 @@ rmarkdown::render(workflow1$reportFileName)
 The output report for Example 1 is shown below. 
 There are only 2 figures in the report (linear and log scales) showing the time profile of the output path '__Organism|A|Concentration in container__' displayed as '__Concentration of A__'.
 
+<div class = "reportOutput">
```
Comment (Yuri05) [inline vignettes/goodness-of-fit.Rmd:146]: applying the style to the report output
@pchelle Theoretically it should also work like this:
`{r example 1 report, echo=FALSE, class.source='reportOutput'}`
But I didn't manage to get it to work then

File: vignettes/reportOutputStyle.css [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<style>
+div.reportOutput { background-color:#e6f0ff; border-radius: 5px; padding: 20px; border-style: solid none solid solid;}
```
Comment (Yuri05) [inline vignettes/reportOutputStyle.css:2]: Style for report outputs: first draft. Report outputs look then like below. We can adjust it to whatever is better looking
grafik

File: vignettes/goodness-of-fit.Rmd (hunk 3 above)
Comment (pchelle) [reply vignettes/goodness-of-fit.Rmd:146]: I think when I tried to look into that I also tried the Rmd code and it did not work on my side neither.
It is maybe not correctly translated into the html tag.
As long as the html tag does the job, I don't mind using it directly.

File: vignettes/reportOutputStyle.css (hunk 4 above)
Comment (pchelle) [reply vignettes/reportOutputStyle.css:2]: That's awesome. 
This is centralized as a .css file + a style class to initialize and call in the vignette using the html tag anytime we need it.

**Review by pchelle — APPROVED** [review]

**Pull request merged**